shipit.prod.yml

```
@@ -0,0 +1,20 @@
+dependencies:
+  override: []
+deploy:
+  override:
+    - git submodule update --init
+    - make install:
+        timeout: 2460
+review:
+  checklist:
+    - Is today not Friday?
+  monitoring:
+    - iframe: https://app.sysdigcloud.com/#/dashboards/222088?last=3600
+      width: 1200
+      height: 800
+ci:
+  hide:
```

```suggestion
  require:
```
the release approval is actually required for prod (and test)